### PR TITLE
New: Add apps minimum seeders per indexer

### PR DESCRIPTION
**Pull request by mynameisbogdan**
#### Database Migration
NO

#### Description
Sometimes the app's minimum seeders value isn't one size fits all for every indexer, so I think this would be a good addition to be able to set it per indexer if you'll like, with fallback to Sync profile's value.

### CODE REVIEW

File: src/NzbDrone.Core/Indexers/IndexerTorrentBaseSettings.cs [hunk 1]
```diff
@@ -38,13 +42,16 @@ public class IndexerTorrentBaseSettings
     {
         private static readonly IndexerTorrentSettingsValidator Validator = new IndexerTorrentSettingsValidator();
 
-        [FieldDefinition(1, Type = FieldType.Textbox, Label = "Seed Ratio", HelpText = "The ratio a torrent should reach before stopping, empty is app's default", Advanced = true)]
+        [FieldDefinition(1, Type = FieldType.Number, Label = "Minimum Seeders", HelpText = "Minimum seeders required for the indexer to grab, empty is app's default", Advanced = true)]
```
Comment (bakerboy448) [inline src/NzbDrone.Core/Indexers/IndexerTorrentBaseSettings.cs:45]: *Sync profile's default

**Comment by Qstick** [comment]
One reason I've been a bit hesitant on this is that it implies that it will apply to all grabs for an indexer, when in reality it only applies to those that come from the apps (i.e. this has no effect on anything outside of app syncing). We should likely clarify that somehow.

**Comment by bakerboy448** [comment]
`Synced App Minimum Seeders` is probably a bit wordy :/

**Review by Qstick — APPROVED** [review]
